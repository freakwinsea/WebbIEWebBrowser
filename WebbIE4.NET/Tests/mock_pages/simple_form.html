<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Simple Form Test</title>
</head>
<body>
    <h1>Login</h1>
    <form action="/login" method="post">
        <div>
            <label for="username">Username:</label>
            <input type="text" id="username" name="user">
        </div>
        <div>
            <label for="password">Password:</label>
            <input type="password" id="password" name="pass">
        </div>
        <div>
            <button type="submit">Log In</button>
        </div>
    </form>
</body>
</html>
