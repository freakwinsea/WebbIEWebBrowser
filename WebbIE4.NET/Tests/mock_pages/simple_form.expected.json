[
  {
    "id": "webbie-id-0",
    "type": "h1",
    "text": "Login",
    "href": ""
  },
  {
    "id": "webbie-id-1",
    "type": "text",
    "text": "Username:",
    "href": ""
  },
  {
    "id": "webbie-id-2",
    "type": "text",
    "text": "",
    "href": ""
  },
  {
    "id": "webbie-id-3",
    "type": "text",
    "text": "Password:",
    "href": ""
  },
  {
    "id": "webbie-id-4",
    "type": "password",
    "text": "",
    "href": ""
  },
  {
    "id": "webbie-id-5",
    "type": "submit",
    "text": "Log In",
    "href": ""
  }
]
